prereceivecli/prereceivecli.py

```diff
@@ -166,6 +189,32 @@ def setup_logging(level, config_file=None):
         logging.getLogger(logger).disabled = True
 
 
+def get_credentials(args):
+    """
+    Gets AWS credentials.
+
+    Needs the args to either assume role or get credentials
+
+    Credentials:
+        Credentials: The AWS credentials to set for our environment
+    """
+    if not all([args.access_key, args.secret_key]):
+        config = Config(retries={'max_attempts': 10,
+                                 'mode': 'standard'})
+        client = boto3.client('sts', config=config)
+        with open(args.aws_web_identity_token_file, 'r') as opened_file:
```

Comment:
This will of course fail if the there is no token file and on the argparser we are not checking that the passed argument is actually a valid file, so this is not really very safe to just assume it would work. We should either check for the existence of the file or check that it is actually read in this part.